app/controllers/mixins/ems_common_angular.rb

```diff
@@ -475,7 +475,7 @@ def get_hostname_from_routes(ems, endpoint_hash, token)
       client = ems.class.raw_connect(endpoint.hostname, endpoint.port,
                                      :service => :openshift, :bearer => token, :ssl_options => ssl_options)
       client.get_route('hawkular-metrics', 'openshift-infra').try(:spec).try(:host)
-    rescue KubeException, OpenSSL::SSL::SSLError => e
+    rescue StandardError => e
```

Comment:
Hmm.  Problem is unexpected exceptions crash UI in a way that doesn't reach user.
Let me make the other PR that reports them better to user, then I'll re-think this rescue...